src/Sulu/Component/Rest/ListBuilder/Metadata/Doctrine/Type/PropertyType.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Metadata\Doctrine\Type;
+
+/**
+ * Abstract base-class for property-type.
+ */
+abstract class PropertyType
```

Comment:
i can also change this to an interface? otherwise i have no possibility to use typehints